webdriverio/index.d.ts

```diff
@@ -352,36 +352,37 @@ declare namespace WebdriverIO {
         ): Client<P>;
         getHTML<P>(includeSelectorTag: boolean): Client<P>;
 
-        getLocation(selector?: string): Size;
-        getLocation(selector: string, axis: string): number;
-        getLocation(axis: string): number;
+        getLocation(selector?: string): Location;
+        getLocation(selector: string, axis: Axis): number;
+        getLocation(axis: Axis): number;
         getLocation<P>(selector?: string): Client<P>;
         getLocation<P>(
             selector: string,
-            axis: string
+            axis: Axis
         ): Client<P>;
-        getLocation<P>(axis: string): Client<P>;
+        getLocation<P>(axis: Axis): Client<P>;
 
-        getLocationInView(selector: string): Size;
-        getLocationInView(selector: string): Size[];
-        getLocationInView(): Size;
-        getLocationInView(): Size[];
+        getLocationInView(selector: string): Location;
+        getLocationInView(selector: string): Location[];
```

Comment:
Hi, I considered that, but I didn't want to introduce breaking changes just for these methods, when the the rest of the Api is written in this way.

I think the whole Api needs to be expressed in a better way, for example:
`client.getLocation(".main")`  => `Location | Location[]`
`client.element(".main").getLocation()`  => `Location`

Because, as I tested, `element` method always returns a single Element.